cmd/scollector/collectors/sql_windows.go

```diff
@@ -499,3 +533,8 @@ type MSCluster_Resource struct {
 	Type       string
 	State      uint32
 }
+
+func instance_wmiquery(instancename string, wmiquery string) string {
```

Comment:
Other than collectors (and that is just a bad convention) funcs should be instanceWMIQuery, unless there is something very similar and you are following the pattern.